### ISSUE TITLE
 'QuerySetFilterJob' object has no attribute 'filter' 

### ISSUE THREAD

**Issue by o3o3o** [codeinthehole/django-cacheback#56]
I use the [example](https://github.com/codeinthehole/django-cacheback/blob/master/docs/api.rst)

```
staff = QuerySetFilterJob(models.User).filter(is_staff=True)
```

When I executed the above code, here is error: 
`
 'QuerySetFilterJob' object has no attribute 'filter' 
`
I think it should be: 

```
QuerySetFilterJob(models.User).get(is_staff=True)
```

**Comment by stephrdev** [comment]
You are right. Thanks for the hint.

**Comment by stephrdev** [comment]
Fixed in f0f4f3b
